File: examples/mvp-java/src/main/java/co/windly/limbosample/presentation/splash/SplashModule.java
package co.windly.limbosample.presentation.splash;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class SplashModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract SplashActivity contibuteAndroidInjector();

  //endregion
}

File: samples/kotlin/src/main/java/co/windly/limbosample/presentation/splash/SplashModule.kt
package co.windly.limbosample.presentation.splash

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class SplashModule {

  //region Contribution

  @ContributesAndroidInjector
  abstract fun contributeAndroidInjector(): SplashActivity

  //endregion
}
